google/cloud/storage/oauth2/google_credentials.cc

```diff
@@ -32,9 +32,21 @@ namespace storage {
 inline namespace STORAGE_CLIENT_NS {
 namespace oauth2 {
 
-/// Parses the JSON file at `path` and creates the appropriate Credentials type.
+constexpr char kAdcLink[] =
+    "https://developers.google.com/identity/protocols/"
+    "application-default-credentials";
+
+/// Parses the JSON or P12 file at `path` and creates the appropriate
```

Comment:
nit: Please use just `//` for this comment.  Doxygen comments with `///` are not useful in `.cc` files, I do realize that the previous comment was wrong already....